core/src/cgcloud/core/generic_boxes.py

```diff
@@ -286,3 +286,44 @@ def __fix_stupid_locale_problem( self ):
         https://www.banym.de/linux/fedora/problems-with-missing-locale-files-on-fedora-20-made-libvirtd-service-not-starting
         """
         sudo( 'localedef -c -i en_US -f UTF-8 en_US.UTF-8' )
+
+
+class GenericEnvironmentPatcherBox( Box ):
```

Comment:
Let's stuff this into RcLocalBox. I have a grand plan for that class to become LSBBox (as in Linux Software Base standard). Then you won't have to mix it in because RcLocalBox is already included in the hierarchy.